scripts/slimy-update-snap.sh

```diff
@@ -53,11 +55,22 @@ do_mount $old old
 add_overlay old new
 
 rm -rf new/lib/python3.10/site-packages/curtin
-rm -rf new/lib/python3.10/site-packages/subiquity
-rm -rf new/lib/python3.10/site-packages/subiquitycore
+
+if [ -d new/lib/python3.10/site-packages/subiquity ] ; then
+    subiquity_dest=new/lib/python3.10/site-packages
+elif [ -d new/bin/subiquity/subiquity ] ; then
+    subiquity_dest=new/bin/subiquity
+else
+    echo what snap is this
+    exit 1
+fi
+
+rm -rf $subiquity_dest/subiquity
+rm -rf $subiquity_dest/subiquitycore
```

Comment:
Let's be extra careful with this kind of command. A space at the wrong place and we can end up with a wiped system.

```bash
rm -rf -- "${subiquity_dest}"/subiquity
rm -rf -- "${subiquity_dest}"/subiquitycore
```

(you can leave aside the `--` if you want)